cmd/bucket-notification-handlers.go

```diff
@@ -144,6 +144,12 @@ func (api objectAPIHandlers) PutBucketNotificationHandler(w http.ResponseWriter,
 		return
 	} // Successfully marshalled notification configuration.
 
+	// Convert the incoming ARNs properly to the GetRegion().
```

Comment:
This comment is not clear. How about the following?
`// Set server's region as the ARN region for all queue notifications configured

Reply:
yes agreed @krisis